aether/init.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2019 by eHealth Africa : http://www.eHealthAfrica.org
```

Comment:
2020? 😄

Reply:
I will change all the copyrights in another PR :grimacing: